bin/youtube-dl.detect-path.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
```

Comment:
While I'm at it: This file is designed to work as a drop-in fix in many projects. If you like it, take it. I hereby grant you permission to use it according to the CC-0 license.